dace/libraries/mpi/utils.py

```diff
@@ -0,0 +1,74 @@
+import dace
+
+
+def MPI_DDT(dtype):
```

Comment:
This is replying to all questions related to this file. I don't really want to touch this file further because there might be further developments soon (but it has to be merged so that the current progress becomes available).